tests/views/test_plot_view.py

```diff
@@ -7,6 +8,8 @@
     select_response,
     wait_a_bit,
 )
+from webviz_ert.plugins import *
```

Comment:
usually not a fan of * imports , any reasons for not being spesific?

Reply:
hmm I've never thought much about the advantages or disadvantages of the `*` import in python, so it might be due to simple lack of critical thinking / knowledge. Basically I just though I want to check all plugins, so I'm gonna import all of them.

i can change it and list the names, if you'd like / think it's better?

Reply:
fixed! the second line in particular seems to have been redundant.